project/src/main/java/com/google/sps/database/ConnectionPoolContextListener.java

```diff
@@ -40,35 +42,66 @@
   private static final String CLOUD_SQL_CONNECTION_NAME =
       System.getenv("CLOUD_SQL_CONNECTION_NAME");
 
+  private static String CREATE_USERS_TABLE = 
+    "CREATE TABLE IF NOT EXISTS users ( "
+    + "id SERIAL NOT NULL, name VARCHAR(255) NOT NULL, email VARCHAR(255) NOT NULL,"
+    + " PRIMARY KEY (id) );";
+
+  private static String CREATE_COMPETITIONS_TABLE = 
+    "CREATE TABLE IF NOT EXISTS competitions ( "
+    + "id SERIAL NOT NULL, start_date DATE NOT NULL, end_date DATE NOT NULL,"
+    + " PRIMARY KEY (id) );";
+
+  private static String CREATE_PARTICIPANTS_TABLE = 
+    "CREATE TABLE IF NOT EXISTS participants ( "
+    + "id SERIAL NOT NULL, user INT NOT NULL, competition INT NOT NULL, amt_available INT NOT NULL,"
+    + " PRIMARY KEY (id) );";
+
+  private static String CREATE_INVESTMENTS_TABLE = 
+    "CREATE TABLE IF NOT EXISTS investments ( "
+    + "id SERIAL NOT NULL, user INT NOT NULL, competition INT NOT NULL, google_search VARCHAR(255) NOT NULL, "
+    + "invest_date DATE NOT NULL, sell_date DATE, amt_invested INT NOT NULL,"
+    + " PRIMARY KEY (id) );";
+
   private static final Logger log = Logger.getLogger(ConnectionPoolContextListener.class.getName());
   @SuppressFBWarnings(
       value = "USBR_UNNECESSARY_STORE_BEFORE_RETURN",
       justification = "Necessary for sample region tag.")
-  private DataSource createConnectionPool() {
+  public DataSource createConnectionPool() {
     // [START cloud_sql_mysql_servlet_create]
     // The configuration object specifies behaviors for the connection pool.
     HikariConfig config = new HikariConfig();
 
     log.info("I'm alive");
 
+    Config mySecrets = new Config();
+
     config.setJdbcUrl(String.format("jdbc:mysql://localhost:3306/%s", "test"));
     config.setUsername("root"); // e.g. "root", "mysql"
-    config.setPassword(REDACTED); // e.g. "my-password"
+    config.setPassword(mySecrets.databasePassword); // e.g. "my-password"
```

Comment:
Can probably remove the example comments now

Reply:
Done